profiles/arch/x86/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Nicolas Werner <nicolas.werner@hotmail.de> (2021-09-05)
+# Qt5 packages don't support multilib, so disable it for this plugin.
+media-plugins/gst-plugins-qt5 abi_x86_64
```

Comment:
x86_64 is really exposed for x86 to need this? Also it's not keyworded x86, so I don't think this mask was added based on repoman complaining

Reply:
Yep, totally not needed, my mind just went to automode there!